wagn/lib/decko/engine.rb

```diff
@@ -12,27 +12,39 @@
 require 'diff/lcs'
 require 'diffy'
 
+require 'wagn'
 
 module Decko
+
   class Engine < ::Rails::Engine
 
     paths.add "app/controllers", :with => 'rails/controllers', :eager_load => true
     paths.add 'gem-assets',      :with => 'rails/assets'
     paths.add 'config/routes',   :with => 'rails/engine-routes.rb'
-    paths.add 'lib/tasks',       :with => 'lib/wagn/tasks', :glob => '**/*.rake'
-    
+    paths.add 'lib/tasks',       :with => "#{::Wagn.gem_root}/lib/wagn/tasks", :glob => '**/*.rake'
+    paths.add 'lib/wagn/config/initializers',
+              :with => File.join( Wagn.gem_root, 'lib/wagn/config/initializers' ), :glob => "**/*.rb"
+
+    initializer 'decko.engine.load_config_initializers',  :after => :load_config_initializers do
+      paths['lib/wagn/config/initializers'].existent.sort.each do |initializer|
+        load(initializer)
+      end
+    end
```

Comment:
I take that back.  I'm enthusiastic but fuzzy. It feels like the default engine setup will be the easiest point of agreement.  Then, after a closer look at the engine docs, I suspect I may have a clearer idea of where we may be thinking differently.

With that reading it looks to me like the standard pattern would be to name the engine Decko::Rails::Engine, no?  Perhaps we define that in decko-rails and have it inherit from Decko::Engine?

One major question I have is where we'll be overriding engine configuration.  In initializers?